ldif_test.go

```diff
@@ -173,7 +175,9 @@ func TestLDIFURL(t *testing.T) {
 	defer os.Remove(f.Name())
 	f.Write([]byte("TEST\n"))
 	f.Sync()
-	l, err := ldif.Parse("dn: uid=someone,dc=example,dc=org\ndescription:< file://" + f.Name() + "\n")
+	d := strings.TrimPrefix(filepath.ToSlash(f.Name()), "C:")
```

Comment:
I appreciate the cleanup; this feels a little brittle with assuming the tempfile will be created in the `C:` volume on windows... what happens if the test runs on windows with a temp dir on the `D:` volume?  Probably not super important in the tests, but something to think about.